test/integration/expect_args_test.rb

```diff
@@ -6,6 +6,8 @@ class ExpectArgsTest < ActionDispatch::IntegrationTest
 
   setup do
     @routes = Oaisys::Engine.routes
+    # Set an action on unpermitted parameters to raise an exception, used to validate parameters.
+    ActionController::Parameters.action_on_unpermitted_parameters = :raise
```

Comment:
This is probably not the best place for this as it will be called once for each test but it's the best place that I could get it working